swap_meet/clothing.py

```diff
@@ -1,2 +1,15 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition=None):
+        if condition == None:
```

Comment:
If we know that the value of condition is supposed to be a float, it can be helpful to hint that with the default parameter.

I see the intent of letting the default condition for Item to be used if the caller leaves off the condition value, but this makes it a little harder for the caller to know what the default condition value is, since now they need to run it back to Item to check.

There may be times when this behavior is exactly what may be desired, in which case, good job! 😄

But in this case, our project description says that the default condition for Item or any of its child classes should be 0.